eo-maven-plugin/src/main/java/org/eolang/maven/MarkMojo.java

```diff
@@ -78,11 +81,22 @@ public void exec() throws IOException {
     private long scan(final Path dir, final String version) {
         final Unplace unplace = new Unplace(dir);
         final Collection<Path> sources = new Walk(dir);
-        final ForeignTojos scoped = this.scopedTojos();
+        final ForeignTojos tojos = this.scopedTojos();
         final long done = sources.stream()
             .filter(src -> src.toString().endsWith(".eo"))
-            .map(unplace::make)
-            .map(scoped::add)
+            .map(
```

Comment:
@maxonfjvipon [optional suggestion] We have lots indents here and code looks a bit [complicated](https://en.wikipedia.org/wiki/Cyclomatic_complexity) now.  Maybe it's better to move it in a separate method with meaningful name?